test/it/parser/src/main/resources/sql/supported/ddl/create-trigger.xml

```diff
@@ -50,4 +50,47 @@
         BEGIN
             :NEW.ID := Contacts_Seq.NEXTVAL;
         END;" db-types="Oracle" />
+    <sql-case id="create_trigger_with_dataManipulationLanguage_statement" value="CREATE OR REPLACE TRIGGER lineitems_trigger
+  AFTER INSERT OR UPDATE OR DELETE ON lineitems
```

Comment:
Is it possible to increase the indentation?

Reply:
I have added appropriate indentation to improve the readability of the code

> Is it possible to increase the indentation?